parameter/DomainConfiguration.cpp

```diff
@@ -397,8 +397,10 @@ void CDomainConfiguration::validate(const CConfigurableElement* pConfigurableEle
 {
     CAreaConfiguration* pAreaConfigurationToValidate = getAreaConfiguration(pConfigurableElement);
 
-    // Delegate
-    pAreaConfigurationToValidate->validate(pMainBlackboard);
+    if (pAreaConfigurationToValidate != NULL) {
```

Comment:
I think that if we abort, or use some kind of "allways_assert". There is some patches in the pipe to introduce such assert. Management was very clear not to introduce workaround for klockwork false positive. Ie adding an assert (or an initialization) is OK but modifying the code flow should be avoided.